docs/Readme.md

```diff
@@ -0,0 +1,23 @@
+# ACS-INTERFACE DOCUMENTATION
+### True for up to commit: f5d8b0c84838e34dd156974d9407831d363b8900
+
+##Steps to set up a new homework:
+
+1. Create a `.ini` file with the name `{classname}_{homework_number}.ini`
+(E.g. `pc_0.ini` for the first homework) in the directory `{root}/assignment/`.
```

Comment:
It would be great if the TA would never log into the vmck or interface server. Let them publish things on github, or the internal gitlab, or wherever. And they can give us a URL when setting up the assignment in the interface.